sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -168,7 +173,13 @@ def __init__(self, feature_id, enabled, filters=[], **kwargs):  # pylint: disabl
         self.etag = kwargs.get("etag", None)
         self.description = kwargs.get("description", None)
         self.display_name = kwargs.get("display_name", None)
-        self.value = kwargs.get("value", {"enabled": enabled, "conditions": {"client_filters": filters}})
+        if "enabled" in kwargs.keys():
+            self.value = kwargs.get(
+                "value",
+                {"enabled": kwargs.pop("enabled"), "conditions": {"client_filters": kwargs.pop("filters", [])}}
```

Comment:
If both "enabled" and "value" are set, what's our behavior?

Reply:
If `value` is set we assume the customer knows what they are doing and we provide no protections.

Reply:
You mean "enabled" will not override the value in "value"? (same for "filters")

Reply:
Correct, if you use value that overrides everything else. You lose any protections these models might provide.